pages/login.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LOGIN | E-Galaxy-store</title>
+    <link rel="stylesheet" href="https://css-ui-galaxy-c.netlify.app/CSS/base.css">
+    <link rel="stylesheet" href="/CSS/login.css">
+    <script src="https://kit.fontawesome.com/8ddd15368f.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="e_container">
+        <nav class="nav_bar">
+            <div>
+                <a href="/index.html">
+                    <h1 class="e_website_name">E-Galaxy-store</h1>
+                    <h5 class="ui_tagline">Find it, love it, buy it.</h5>
+                </a>
+
+            </div>
+            <div class="search">
+                <button>
+                    <i class="fa fa-search"></i>
+                </button>
+                <input type="text" placeholder=" Search Your Desier" name="search">
+            </div>
+            <div>
+                <ul class="nav_list">
+                    <li><a class="product_btn" href="/">Products</a></li>
+                    <li><a class="login_btn" href="/pages/login.html">Login</a></li>
+                    <li class="list-item-inline"><a href="/"><i class="icon_nav fa fa-heart-o"></i></a></li>
+                    <li class="list-item-inline"><a href="/"><i class="icon_nav fas fa-shopping-cart"></i></a>
+                    </li>
+                    <li class="list-item-inline"><a href="/"><i class="icon_nav fas fa-user"></i></a></li>
+                </ul>
+            </div>
+        </nav> 
+        <div class="login_container">
+            div.
+        </div>
+        <footer>
+            <footer class="footer">
+                <div class="footer-header">
+                    <p><a href="/">Terms & Conditions</a></p>
+                    <p><a href="/">Contact Us</a></p>
+                </div>
+                <ul class="social_media_links">
```

Comment:
you can add target attributes in all social media  anchor tag , so that it will open in new tab

Reply:
okay i will do that thank you